app/services/gov_delivery/request_builder.rb

```diff
@@ -35,10 +35,14 @@ def self.send_bulletin_xml(topic_ids, subject, body, options = {})
             xml.cdata options[:header]
           } if options[:header]
           xml.footer {
-            xml.cdata options[:footer]
-          } if options[:footer]
+            xml.cdata options[:footer] || default_footer
+          }
         }
       }.to_xml
     end
+
+    def self.default_footer
+      @default_footer ||= File.read(File.expand_path(File.join(__dir__, 'templates', 'default_footer.html')))
```

Comment:
Was wondering if it's cleaner to use rails views for this, rather than adding a templates dir. I'm fine with either approach though.